src/pages/docs/git.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Git / GitHub Workflow for CAOS
+description: A tutorial on how to use Git and GitHub to contribute to CAOS
+---
+
+A tutorial on how to use Git and GitHub to contribute to CAOS
+
+---
+# Git / GitHub Workflow for CAOS
+
+Git is a source version control system created by Linus Torvalds in 2005. This guide will walk you through configuring it so that you can start contributing to CAOS. Git is a system that works on repositories: a git repository contains all the source code of your application, as well as its history, i.e., all previous versions of it. These repositories need to be hosted somewhere, in order to be accessible by everyone. CeSIUM chose [GitHub](https://github.com/cesium) as its host, however, different hosts exist.
+
+
+## Configuring your GitHub Account
+
+The first step is to create a GitHub account. Go to [GitHub](https://github.com) and create your account.
+
+### SSH Key
+
+After your account is created, you need to configure your SSH keys in order to be able to authenticate whenever you upload your code. An [SSH key](https://www.ssh.com/academy/ssh-keys) is an access credential that will allow git to login in GitHub without you having to input your email and password - which, as an authentication method, has been removed by GitHub altogether. To generate an SSH key, go to your terminal and type
+
+```
+ssh-keygen
```

Comment:
```suggestion
ssh-keygen -t ed25519 -C "your_email@example.com"
```

Reply:
I requested this change in order to connect with what they ask on Github

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent